spec/helpers/glossary_helper_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.describe GlossaryHelper do
+  def sanitize(doc)
+    doc.tap{ doc.css('img').each(&:remove) }
+  end
+
+  describe '#link_glossary_terms' do
+    it 'should do nothing when the document contains no terms' do
```

Comment:
nit: use `let`s and `context`s instead of assigning variables in the example, especially when variables are used in multiple expectations